lib/twitter/rest/direct_messages.rb

```diff
@@ -120,7 +123,7 @@ def destroy_direct_message(*args)
       # @raise [Twitter::Error::Unauthorized] Error raised when supplied user credentials are not valid.
       # @return [Twitter::DirectMessage] The sent message.
       # @param user [Integer, String, Twitter::User] A Twitter user ID, screen name, URI, or object.
-      # @param text [String] The text of your direct message, up to 140 characters.
+      # @param text [String] The text of your direct message, up to 10K characters.
```

Comment:
Is this `10,000` or `10 * 2 ** 10`? Either way, it’s probably clearer to spell it out.